nix/driverBuildScript.nix

```diff
@@ -0,0 +1,71 @@
+# Shell script to build the driver.
+# It's available inside the default development shell,
+# and is also used by the cross-build shells (crossBuild.nix).
+# It is meant to be called from the Makefile.
+#
+# Usage: build-driver -v <version> -i <input> -o <output>
+# See Makefile for concrete examples.
+
+{ pkgs
+, GOOS
+, GOARCH
+, CC
+, staticBuild
+}:
+let
+  staticLinkingLdFlags =
+    if staticBuild then
+      "-linkmode external -extldflags '-static'"
+    else
+      "";
+in
+(pkgs.writeShellApplication {
```

Comment:
Could this just be a shell script?

I might be missing something, but it looks like the parameters could just be passed as CLI flags or env variables? Do we need to write a script file with Nix at all?

Reply:
You're right, it doesn't have to be a nix script:
[Use regular shell script instead of nix generated script](https://github.com/dividat/driver/pull/129/commits/5eabfa62a5f3f82fb8486accf2e28b3d27ca723c)